src/redux/data.js

```diff
@@ -261,10 +261,7 @@ export const addressAssetsReceived = (
   const liquidityTokens = remove(
     assets,
     asset =>
-      asset?.asset?.type === 'uniswap' ||
-      // Remove duplicate liquidity tokens when falling back from zerion
-      (toLower(asset?.asset.name).indexOf('uniswap') !== -1 &&
-        toLower(asset?.asset.name) !== 'uniswap')
+      asset?.asset?.type === 'uniswap' || asset?.asset?.type === 'uniswap-v2'
```

Comment:
Just a note: I removed the extra check here for the fallback from Zerion bc now I parse out and include the uniswap LP token type in the fallback explorer discover tokens logic